js/src/lib/components/InferenceWidget/shared/helpers.ts

```diff
@@ -180,8 +183,8 @@ export async function getResponse<T>(
 }
 
 
-export async function getModelLoadInfo(url: string, repoId: string, includeCredentials = false): Promise<ModelLoadInfo> {
-	const response = await fetch(`${url}/status/${repoId}`, {credentials: includeCredentials ? "include" : "same-origin"});
+export async function getModelLoadInfo(url: string, repoId: string): Promise<ModelLoadInfo> {
+	const response = await fetch(`${url}/status/${repoId}`);
```

Comment:
are you undoing this change from #889 on purpose, and if so, are you sure about it? cc @mishig25

Reply:
It may be a merging issue, I'm going to look up.

Reply:
Fixed !